caffe2/contrib/tensorrt/tensorrt_op_trt.cc

```diff
@@ -0,0 +1,240 @@
+/**
+ * Copyright (c) 2016-present, Facebook, Inc.
+ *
```

Comment:
Can you remove this header? We don't put headers on the files anymore and rely on the base LICENSE file instead. This is PyTorch convention.